concourse/tasks/daisy-build-images-windows.yaml

```diff
@@ -0,0 +1,32 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/compute-image-tools/daisy
+    tag: latest
+
+inputs:
+- name: compute-image-tools
+- name: credentials
+- name: publish-version
+
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: "credentials/credentials.json"
+
+run:
+  path: /daisy
+  args:
+  - -project=gce-image-builder
+  - -zone=us-central1-c
+  - -var:media=((iso))
+  - -var:pwsh=((pwsh))
+  - -var:dotnet48=((dotnet48))
+  - -var:cloudsdk=((cloudsdk))
+  - -var:updates=((updates))
+  - -var:build_date=((build_date))
+  - -var:google_cloud_repo=((google_cloud_repo))
+  - -var:gcs_url=((gcs_url))
+  - -var:gcs_interal_url=((gcs_internal_url))
```

Comment:
Where does gcs_internal_url come from?

Reply:
Ah nm- found it. I think it just needs to be plumbed into the rest of the tasks.